src/main/java/com/cburch/logisim/gui/start/Startup.java

```diff
@@ -95,7 +97,11 @@ public class Startup implements AWTEventListener {
   private String circuitToTest = null;
   private boolean exitAfterStartup = false;
   private boolean showSplash;
-  private File loadFile;
+
+  /* File contains the data that should be loaded into a RAM/ROM with a label matching the String
+     (if no label is provided, File is loaded into every RAM/ROM) */
+  private List<Pair<File, String>> memoriesToLoad = new ArrayList<>();
```

Comment:
This should be also reflected in the user hints and documentation!

Reply:
Sorry about missing to update the documentation.

Question: how shall I handle languages other than English?

While I am happy to do it for English and French as I am fluent in these languages, what about the rest? ChatGPT/Google Translate? Or is there a way to notify/mark the fact that a translation is no longer up to date? Will the pull request be on hold until all translations have been done?

I couldn't find any information about this in the "How to contribute" section.

Reply:
> While I am happy to do it for English and French as I am fluent in these languages, what about the rest?

I'd say just update the languages you speak. Automated translation is usually fine as long as we have a proofreader, otherwise it can end up with nonsense esp. we talk technical language.

Reply:
I agree with @MarcinOrlowski, please take care of the languages you speak. In the other language versions, you may add a TODO marker (in English) drawing attention that it is outdated, but you don't have to proactively do automatic translation. The PR won't be put on hold due to missing translations.

@mbaillif may be able to give further suggestions regarding documentation, as he has contributed most improvements to it over the past years.